### DISCUSSION TITLE
OSError: No socket could be created -- (('192.168.1.144', 8080): [Errno 99] Cannot assign requested address)

### DISCUSSION THREAD

**Discussion by OldSurferDude** [Dan-in-CA/SIP#296] [Q&A] [answered]
In [OPTIONS]-->system I set up HTTP port and HTTP IPaddr.  The form I left the port at 80 and the IPaddr set to the  IP address of the RPI.  I did some configurations, some testing and then rebooted.

But SIP did not start.  I edited the file /opt/SIP/data/sd.json and changed the "htp" to 8080 and started sip with
`sudo systemctl start sip
`
Even if I don't start SIP, if I reboot with the new port, it will not start automatically.  This is a problem because one has to anticipate power failures.

Does anyone have any ideas?

OSD

**Comment by Dan-in-CA** [comment, marked as answer]
The "OSError: No socket could be created" message usually indicates that something is already using the port.
the HTTP IP addr:  field should be :: (double colon)
